### PR TITLE
Add multiple coins

### PR DESCRIPTION
**Pull request by onnateldome**
Added some coins along with tests

### CODE REVIEW

File: electrumx/lib/coins.py [hunk 1]
```diff
@@ -3842,3 +3842,198 @@ class Quebecoin(AuxPowMixin, Coin):
...
+    TX_PER_BLOCK = 1
+    RPC_PORT = 9323
+    SESSIONCLS = DashElectrumX
+    DAEMON = daemon.DashDaemon
+    REORG_LIMIT = 8000
+
+    @classmethod
+    def header_hash(cls, header):
+        import neoscrypt
+        return neoscrypt.getPoWHash(header)
```
Comment (JustinTArthur) [inline electrumx/lib/coins.py:4039]: I'm afraid we aren't able to take on neoscrypt coins until sparkspay/python-neoscrypt#1 has been resolved.

File: electrumx/lib/coins.py [hunk 2]
```diff
@@ -3842,3 +3842,198 @@ class Quebecoin(AuxPowMixin, Coin):
...
+    TX_COUNT_HEIGHT = 1253018
+    TX_PER_BLOCK = 2
+    RPC_PORT = 20102
+    REORG_LIMIT = 5000
+    HEADER_HASH = None
+
+    @classmethod
+    def header_hash(cls, header):
+        if cls.HEADER_HASH is None:
+            import scrypt
```
Comment (JustinTArthur) [inline electrumx/lib/coins.py:4014]: We're using stdlib hashlib.scrypt now.

**Comment by gits7r** [comment]
I recommend to reject this PR and close. This particular ElectrumX is focused on Bitcoin only, and we should focus on supporting Bitcoin (BTC) further, maintaining and fixing as well as adding new features. Adding altcoins only complicates our job. Any altcoin can fork their own version directly from upstream and support their respective coins but adding altcoins to this particular repo should be discouraged.

**Comment by ahmedbodi** [comment]
@gits7r, since thats the stance, would you be okay with either me or yourself just replying to these and redirecting them to here: https://github.com/electrum-altcoin/electrumx. May aswell isolate all the altcoin PR's into 1 repo that wants them?

**Pull request closed without merging**